worker/Makefile

```diff
@@ -101,3 +101,7 @@ docker-run:
 		--cap-add SYS_PTRACE \
 		-v $(shell pwd)/../:/mediasoup \
 		mediasoup/docker:latest
+
+libmediasoup-worker:
```

Comment:
Do we have to call `make libmediasoup-worker` manually? or how is it called? If it makes sense to call it manually, can you document it in `doc/Building.md`?

Reply:
Yes, defaults didn't change, needs to be called manually. Added to the doc.

Reply:
But somewhere else? Otherwise "npm install mediasoup" won't build it. See npm-scripts.js please.

Reply:
You wouldn't use it with NPM.

The primary use case I see right now it to call it from build script in Rust crate, see: https://github.com/nazar-pc/mediasoup/blob/a67a5ad535315979e273644a9f215ac24c79e451/worker/build.rs#L5-L15